app/full_text_search.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import re
+
+from google.appengine.api import search as appengine_search
+
+import model
+
+# The index name for full text search
+PERSON_FULLTEXT_INDEX_NAME = 'person_information'
+
+def search(repo, query_obj, max_results):
```

Comment:
query_obj should be query_txt or something. query_obj sounds like not a simple string e.g., an object of TextQuery.